pkg/network/types/types_test.go

```diff
@@ -86,6 +86,36 @@ func TestDumpIPNet(t *testing.T) {
 	}
 }
 
+func TestDumpMacaddress(t *testing.T) {
+	mustParse := func(in string) schema.MacAddress {
+		mac, err := net.ParseMAC(in)
+		if err != nil {
```

Comment:
this is just a test, no need to panic
```suggestion
              require.NoError(t,err)
```